src/entities/notification/model/factory/tests/NotificationBuilder.daily.test.ts

```diff
@@ -0,0 +1,617 @@
+        scheduleEvent: event,
+      };
+
+      expect(result.events).toEqual([expectedResult]);
+      expect(result.events[0].notifications.length).toEqual(9);
+    });
+
+    it('Should return array of 24 notifications including reminders when reminder is set and activityIncomplete is 1 and event-endDate is today + 7 days', () => {
+      const today = new Date(2024, 0, 3);
+      const now = addTime({ hours: 15, minutes: 30 }, today);
+
+      const event = getEmptyEvent();
+      setNormalSettingsToEvent(event, PeriodicityType.Daily, today, true);
+
+      event.availability.endDate = addDays(today, 7);
+
+      event.scheduledAt = calculateScheduledAt(event, now);
+
+      const eventEntity = getEventEntity(event);
+
+      const builder = createBuilder(eventEntity);
+      mockUtilityProps(builder, now);
+
+      const result = builder.build();
+
+      const expected: NotificationDescriber[] = [];
+
+      addReminder(expected, false, subDays(today, 7), 1, 'invalid-period');
+      addReminder(expected, false, subDays(today, 6), 1, 'invalid-period');
+      addReminder(expected, false, subDays(today, 5), 1, 'invalid-period');
+      addReminder(expected, false, subDays(today, 4), 1, 'invalid-period');
+      addReminder(expected, false, subDays(today, 3), 1, 'invalid-period');
+      addReminder(expected, false, subDays(today, 2), 1, 'outdated');
+
+      addNotification(expected, subDays(today, 1), 'outdated');
+      addReminder(expected, false, subDays(today, 1), 1);
+      addNotification(expected, new Date(today));
+      addReminder(expected, false, new Date(today), 1);
+      addNotification(expected, addDays(today, 1));
+      addReminder(expected, false, addDays(today, 1), 1);
+      addNotification(expected, addDays(today, 2));
+      addReminder(expected, false, addDays(today, 2), 1);
+      addNotification(expected, addDays(today, 3));
+      addReminder(expected, false, addDays(today, 3), 1);
+      addNotification(expected, addDays(today, 4));
+      addReminder(expected, false, addDays(today, 4), 1);
+      addNotification(expected, addDays(today, 5));
+      addReminder(expected, false, addDays(today, 5), 1);
+      addNotification(expected, addDays(today, 6));
+      addReminder(expected, false, addDays(today, 6), 1);
+      addNotification(expected, addDays(today, 7));
+      addReminder(expected, false, addDays(today, 7), 1, 'invalid-period');
+
+      const expectedResult: EventNotificationDescribers = {
+        eventId: 'mock-event-id',
+        eventName: 'For mock-entity-name, DAILY, 1 notifications, reminder set',
+        notifications: expected,
+        scheduleEvent: event,
+      };
+
+      expect(result.events).toEqual([expectedResult]);
+      expect(result.events[0].notifications.length).toEqual(24);
+    });
+
+    it('Should return array of 18 notifications including reminders when reminder is set and activityIncomplete is 1 and event-startDate is today + 5 days', () => {
```

Comment:
I am not sure what "activityIncomplete is 1" means.

Does it mean this reminder?:
`addReminder(expected, false, addDays(today, 13), 1, 'invalid-period');`